LocalStoreSimulation/src/ro/scit/menu/Menu.java

```diff
@@ -0,0 +1,146 @@
+                        }
+                        System.out.println("Enter the price of the product: ");
+                        int price = sc.nextInt();
+                        System.out.println("Enter the weight of the product: ");
+                        int weight = sc.nextInt();
+                        System.out.println("Enter the quantity of the product: ");
+                        int quantity = sc.nextInt();
+                        System.out.println("Enter the validity date  with format yyyy-MM-dd format:");
+                        String validityDate = sc.next();
+                        System.out.println("Enter the storage temperature: ");
+                        int storageTemperature = sc.nextInt();
+                            productList.add(new Animal(id, animalName, price, weight, quantity, validityDate, storageTemperature));
+                            System.out.println("Your product is in stock! ");
+                        // Vegetable product
+                    } else if (type == 2) {
+                        int id = 0;
+                        Vegetable vegetable = new Vegetable();
+                        vegetable.setType("VEGETABLE");
+                        System.out.println("Enter the name of the vegetable: ");
+                        String name = sc.next();
+                        System.out.println("Enter the price of the product: ");
+                        int price = sc.nextInt();
+                        System.out.println("Enter the weight of the product: ");
+                        int weight = sc.nextInt();
+                        System.out.println("Enter the quantity of the product: ");
+                        int quantity = sc.nextInt();
+                        System.out.println("Enter the validity date with format yyyy-MM-dd format");
+                        String validityDate = sc.next();
+                        System.out.println("Enter list of vitamins: ");
+                        String vitaminList = sc.next();
+                        productList.add(new Vegetable(id, name, price, weight, quantity, validityDate, vitaminList));
+                        System.out.println("Your product is in stock! ");
+                    } else {
+                        System.out.println("Invalid option! Please try again!");
+                    }
+                    break;
+                case 2:
+                    productList.forEach(System.out::println);
+                    System.out.println("=================================");
+                    System.out.println("Insert the date with format yyyy-MM-dd format ");
+                    String sellDate = sc.next();
+                    System.out.println("Insert the id of the product: ");
+                    int id = sc.nextInt();
+                        System.out.println("Insert the quantity: ");
+                        int soldQuantity = sc.nextInt();
+                        int quantity = product.getQuantity();
```

Comment:
what product is this? you don't get it based on id... it's the one you did with new Product before the switch...